mlir/include/mlir/Dialect/MIOpen/gridwise_gemm_params.h

```diff
@@ -614,14 +675,23 @@ class TunableParameters {
   std::map<std::string, int> params;
 };
 
-struct InitParamsNonXDL : InitParams {
+struct InitParamsNonXDL : InitParams, Serializable<InitParamsNonXDL> {
```

Comment:
how would XDLOPS-related tuning parameters be set?

Reply:
Unfortunately InitParamsXDL is not there yet. (All my focuses have been on non-XDLOps, cpp backend.) I can work on this as next piece of work: Add tuning parameters generation logic to XDLOPs.

Reply:
sounds good. let's make this the highest priority after this PR.